website/src/main/tut/docs/users/installation.md

```diff
@@ -21,15 +28,16 @@ addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "{{ site.stableVersion }}")
 // ===> sbt shell
 > scalafixEnable // Setup scalafix for active session.
                  // Not needed if build.sbt is configured like below.
-> scalafixCli                                            // Run all rules configured in .scalafix.conf
-> scalafixCli --rule RemoveUnusedImports                 // Run only RemoveUnusedImports rule
-> myProject/scalafixCli --rule RemoveUnusedImports       // Run rule in one project only
-> test:scalafixCli --rule RemoveUnusedImports            // Run rule in single configuration
-> scalafixCli --rule ExplicitR<TAB>                      // Use tab completion
-> scalafixCli --rule replace:com.foobar/com.buzbaz       // Refactor (experimental)
-> scalafixCli --rule file:rules/MyRule.scala             // Run local custom rule
-> scalafixCli --rule github:org/repo/v1                  // Run library migration rule
-> scalafixCli --test                                     // Make sure no files needs to be fixed
+> compile:scalafixCli                                            // Run all rules configured in .scalafix.conf 
```

Comment:
Is the `compile:` prefix necessary? I would expect it to default to `compile:`

Reply:
Yeah, you're right :)